scripts/geome_things.py

```diff
@@ -169,21 +176,31 @@ def main(ctx, db_url, solr_url, verbosity):
     default=1000,
     help="Maximum records to load, -1 for all",
 )
+@click.option(
+    "-p",
+    "--project_id",
+    type=int,
+    default=0,
+    help="The project id to fetch records for",
+)
```

Comment:
I wanted to be able to test the harvest path easily.  Plumbing in a project id parameter let me just fetch all records for a specific GEOME project.  From there, I could verify they got written to the database.